aioftp/client.py

```diff
@@ -1092,18 +1094,24 @@ async def get_passive_connection(self, conn_type="I",
             "epsv": self._do_epsv,
             "pasv": self._do_pasv,
         }
-        if not commands:
+        if not commands and not self._passive_commands:
             raise ValueError("No passive commands provided")
+
+        passive_commands = self._passive_commands
+        if commands is not None:
+            # Overwrite the default passive commands
+            passive_commands = commands
```

Comment:
Strange logic. I think it is better to write something like
``` python
if not commands:
    commands = self._passive_commands
if not commands:
    raise ValueError(...)
```

Reply:
Good point. I've changed it.